typescript/sdk/src/middleware/liquidity-layer/LiquidityLayerApp.ts

```diff
@@ -141,7 +146,11 @@ export class LiquidityLayerApp extends HyperlaneApp<
           try {
             return [CircleBridgeAdapterInterface.parseLog(_)];
           } catch {
-            return [];
+            try {
+              return [MailboxInterface.parseLog(_)];
```

Comment:
Nit: underscore is a bad var name because it's typically used to imply the var isn't used

Reply:
I'll do this since this is not a pattern in TS, but _ are often used as the single function arg that takes the least visual space
https://stackoverflow.com/questions/10538924/c-sharp-style-lambdas-or-x
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1717
https://www.baeldung.com/scala/underscore#:~:text=The%20underscore%20can%20be%20used,used%20anywhere%20in%20the%20code.